docs/guides/cloud/github-integration.mdx

```diff
@@ -375,6 +375,17 @@ integration settings:
   alt="Status checks settings"
 />
 
+:::tip
+
+<strong>
+  <Icon name="star" /> Premium Cypress Cloud Feature
+</strong>
+
+Reporting on flakey status checks is included in our
+[Team, Business and Enterprise paid pricing plans](https://www.cypress.io/pricing).
+
+:::
+
 ## Pull request comments
```

Comment:
I realize that the bullet lists under here are out of date

Reply:
Took a stab at updating the bullets to what I could gather. Let me know if you want something more specific.